tests/Doctrine/Tests/ORM/Functional/Ticket/DDC3785Test.php

```diff
@@ -31,18 +31,23 @@ protected function setUp()
      */
     public function testOwningValueObjectIdIsCorrectlyTransformedWhenRemovingOrphanedChildEntities()
     {
-    	$id = new DDC3785_AssetId("919609ba-57d9-4a13-be1d-d202521e858a");
-    	$attributes = [
-    		$attribute1 = new DDC3785_Attribute("foo1", "bar1"),
-    		$attribute2 = new DDC3785_Attribute("foo2", "bar2")
+        $id = new DDC3785_AssetId('919609ba-57d9-4a13-be1d-d202521e858a');
+
+        $attributes = [
+            $attribute1 = new DDC3785_Attribute('foo1', 'bar1'),
+            $attribute2 = new DDC3785_Attribute('foo2', 'bar2')
         ];
+
         $this->_em->persist($asset = new DDC3785_Asset($id, $attributes));
         $this->_em->flush();
 
-        $asset->getAttributes()->removeElement($attribute1);
+        $asset->getAttributes()
+              ->removeElement($attribute1);
 
         $this->_em->persist($asset);
         $this->_em->flush();
+
+        $this->addToAssertionCount(1);
```

Comment:
Test works on orphan removal: we should attempt a lookup on the orphaned element